android/logto-client/src/main/kotlin/io/logto/sdk/android/LogtoClient.kt

```diff
@@ -1,3 +1,19 @@
 package io.logto.sdk.android
 
-class LogtoClient
+import io.logto.sdk.android.type.AccessToken
+import io.logto.sdk.android.type.LogtoConfig
+import io.logto.sdk.core.type.OidcConfigResponse
+
+open class LogtoClient(
+    val logtoConfig: LogtoConfig
+) {
+    protected val accessTokenMap: Map<String, AccessToken> = mutableMapOf()
+
+    protected var refreshToken: String? = null
+
+    protected var idToken: String? = null
+
+    protected var oidcConfig: OidcConfigResponse? = null
```

Comment:
do we need explicit init for optional values?

Reply:
This is required in kotlin